src/controllers/update/update.beneficial.owner.statements.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { NextFunction, Request, Response } from "express";
+import { Session } from "@companieshouse/node-session-handler";
+
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+import { ApplicationData } from "../../model";
+import { checkBOsDetailsEntered, checkMOsDetailsEntered, getApplicationData, setExtraData } from "../../utils/application.data";
+import { BeneficialOwnersStatementType, BeneficialOwnerStatementKey } from "../../model/beneficial.owner.statement.model";
+import { saveAndContinue } from "../../utils/save.and.continue";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_PAGE}`);
```

Comment:
Please move get() and post() bodies to a common file to share with registration. I think only the URL links are changing.

Reply:
Done